config/default.json

```diff
@@ -26,15 +26,15 @@
         "persistDuplicateCache": false
     },
     "logger": {
-        "consoleLogLevel": "verbose",
+        "consoleLogLevel": "warn",
         "logLevel": "verbose",
         "enableLogs": {
-          "webhooks": true,
+          "webhooks": false,
           "discord": true,
           "telegram": true
         },
         "dailyLogLimit": 7,
-        "webhookLogLimit": 12
+        "webhookLogLimit": 2
```

Comment:
Not sure why a reduction is even needed if the default is to be off (which I guess I can agree with given there could be a lot of disk space used unexpectedly)

Reply:
Well, if someone enables it, he probably still doesn't need 12x 1,5GB of webhook logs, does he? It's just prone to filling the disk.

Reply:
If they are turning it on, they have a reason. Often it is to capture and understand quests or find a specific condition. If they are going to the extent of overriding it they can consider how many hours they get.
Size of the log is dependent on your system of course.